jobs/config_test.py

```diff
@@ -628,10 +628,6 @@ def test_valid_job_config_json(self):
                                   'both set or unset: %s' % job)
 
                     if job.startswith('pull-kubernetes-') and not node_e2e:
-                        if not 'pull-kubernetes-federation-e2e-gce' in job:
```

Comment:
@krzyzacy maybe we need a TODO to start doing similar checks properly outside of k/k?
(again, not related to this PR)